### ISSUE TITLE
Need a maintainer

### ISSUE THREAD

**Issue by Ceasar** [Ceasar/staticjinja#82]
Hi all,

I'm glad many people are finding this project useful. I do not have time to maintain this project any more and would like to transfer maintenance over to someone else.

If you can step up, please contact me so we can talk about transferring permissions. Either here, or privately at my email: cbautista2010@gmail.com.

**Comment by PatrickMassot** [comment]
Hi @Ceasar, I'm willing to maintain this project.

**Comment by NickCrews** [comment]
Hey @PatrickMassot  are you still interested? I just reached out again and Caesar gave me write permissions, would you want to help me maintain this?

**Comment by PatrickMassot** [comment]
@NickCrews thank you very much for taking up maintenance, and for asking me. I currently maintain many more projects than when I offered to help. And already my initial offer was only because I found it really sad to see this nice little project abandoned. See also my comment [here](https://github.com/Ceasar/staticjinja/pull/65#issuecomment-619414669).

So it wouldn't be reasonable to say that I can become an official maintainer here. But I'll certainly stay subscribed to notifications from this project. I hope I'll be able to contribute comments from time to time.
